README.md

```diff
@@ -3,6 +3,7 @@
 ![License](https://img.shields.io/crates/l/cggmp21.svg)
 [![Docs](https://docs.rs/cggmp21/badge.svg)](https://docs.rs/cggmp21)
 [![Crates io](https://img.shields.io/crates/v/cggmp21.svg)](https://crates.io/crates/cggmp21)
+[![Discord](https://img.shields.io/discord/905194001349627914?logo=discord&logoColor=ffffff&label=Discord)][in Discord]
```

Comment:
What does the part in the square brackets at the end of the line do?

Reply:
It’s a link, it’s just defined in the end of the doc

Reply:
Here:

https://github.com/LFDT-Lockness/cggmp21/blob/f7574c6c488543cbc952a4c5dfc34cc9d0e3521d/README.md?plain=1#L265-L265

Reply:
Ah I see